_posts/2023-01-15-release-table-store-0.3.0.md

```diff
@@ -0,0 +1,238 @@
+
+This feature is only suitable for tables containing at most tens of millions of records to avoid excessive use of local disks.
+
+## Time Traveling
+
+You can use [Snapshots Table]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/how-to/querying-tables/#snapshots-table)
+and [Scan Mode]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/how-to/querying-tables/#scan-mode) to time traveling.
+
+- You can view all current snapshots by reading snapshots table.
+- You can travel time through `from-timestamp` and `from-snapshot`.
+
+For example:
+
+```
+SELECT * FROM T$options;
+
+/*
++--------------+------------+--------------+-------------------------+
+|  snapshot_id |  schema_id |  commit_kind |             commit_time |
++--------------+------------+--------------+-------------------------+
+|            2 |          0 |       APPEND | 2022-10-26 11:44:15.600 |
+|            1 |          0 |       APPEND | 2022-10-26 11:44:15.148 |
++--------------+------------+--------------+-------------------------+
+2 rows in set
+*/
+
+SELECT * FROM T /*+ OPTIONS('from-snapshot'='1') */;
+```
+
+## Audit Log Table
+
+If you need to audit the changelog of the table, you can use the
+[audit_log]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/how-to/querying-tables/#audit-log-table)
+system table. Through audit_log table, you can get the `rowkind` column when you get the incremental data of the table.
+You can use this column for filtering and other operations to complete the audit.
+
+There are four values for `rowkind`:
+
+- +I: Insertion operation.
+- -U: Update operation with the previous content of the updated row.
+- +U: Update operation with new content of the updated row.
+- -D: Deletion operation.
+
+For example:
+
+```
+SELECT * FROM MyTable$audit_log;
+
+/*
++------------------+-----------------+-----------------+
+|     rowkind      |     column_0    |     column_1    |
++------------------+-----------------+-----------------+
+|        +I        |      ...        |      ...        |
++------------------+-----------------+-----------------+
+|        -U        |      ...        |      ...        |
++------------------+-----------------+-----------------+
+|        +U        |      ...        |      ...        |
++------------------+-----------------+-----------------+
+3 rows in set
+*/
+```
+
+## Ecosystem
+
+Flink Table Store continues to strengthen its ecosystem and gradually gets through the reading and writing
+of all engines. Each engine below 0.3 has been enhanced.
+
+- Spark write has been supported. But `INSERT OVERWRITE` and stream write are still supported.
+- [S3]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/filesystems/s3/) and
+  [OSS]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.3/docs/filesystems/oss/)
+  are supported by all computing engines.
+- Hive 3.1 are supported.
+- Trino High versions (JDK 1.17) are supported.
```

Comment:
Why is the "High" capitalized? Is "Trino High" refers to specific terminology? Would be better to clarify what more versions are supported.

Reply:
I will change this to JDK 1.17 versions

Reply:
Currently, the api has not changed, and the new version can support it